hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -2300,6 +2374,14 @@ func (r *NodePoolReconciler) getNodePoolNamespacedName(nodePoolName string, cont
 	return types.NamespacedName{Name: nodePoolName, Namespace: nodePoolNamespace}, nil
 }
 
+func isNodePoolGeneratedConfigMap(cm *corev1.ConfigMap) bool {
```

Comment:
no so obvious to infer which ConfigMap, maybe `isNodePoolGeneratedTuningConfigMap` ?